package.json

```diff
@@ -20,36 +19,27 @@
     "test:unit": "yarn test --selectProjects UNIT"
   },
   "devDependencies": {
-    "@babel/core": "^7.11.4",
-    "@babel/eslint-parser": "^7.11.4",
-    "@babel/plugin-proposal-optional-chaining": "^7.12.1",
-    "@babel/plugin-syntax-flow": "^7.10.4",
-    "@babel/plugin-transform-flow-strip-types": "^7.10.4",
-    "@babel/preset-env": "^7.11.0",
-    "@babel/register": "^7.10.5",
-    "@comicrelief/eslint-config": "^1.2.0",
-    "babel-loader": "^8.1.0",
-    "babel-plugin-module-resolver": "^4.0.0",
-    "dotenv": "^8.2.0",
-    "eslint": "^7.7.0",
-    "eslint-config-airbnb-base": "^14.2.0",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-import-resolver-babel-module": "^5.1.2",
-    "eslint-plugin-flowtype": "^5.2.0",
-    "eslint-plugin-import": "^2.3.0",
-    "eslint-plugin-prettier": "^3.1.4",
-    "eslint-plugin-sonarjs": "^0.5.0",
-    "eslint-plugin-unicorn": "^21.0.0",
+    "@comicrelief/eslint-config": "^2.0.3",
+    "@types/jest": "^29.5.1",
+    "@types/node": "^18.15.12",
+    "@typescript-eslint/eslint-plugin": "^5.59.0",
+    "@typescript-eslint/parser": "^5.59.0",
+    "eslint": "^8.38.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-jsdoc": "^43.0.6",
     "jest": "^29.5.0",
     "prettier": "^2.1.1",
     "serverless": "3",
     "serverless-offline": "12.0.4",
-    "serverless-webpack": "^5.3.4",
-    "webpack": "^4.44.1",
-    "webpack-node-externals": "^2.5.2"
+    "serverless-plugin-typescript": "^2.1.4",
+    "ts-jest": "^29.1.0",
+    "ts-node": "^10.9.1",
+    "tsconfig-paths": "^4.2.0",
+    "typescript": "^5.0.4"
   },
   "dependencies": {
-    "@comicrelief/lambda-wrapper": "^1.1.10",
-    "axios": "^1.3.6"
+    "@comicrelief/lambda-wrapper": "^2.0.0-beta.3",
+    "axios": "^1.3.6",
+    "dotenv": "^8.2.0"
```

Comment:
Shouldn't `dotenv` stay in `devDependencies`?

Reply:
Yup, you're right. No idea why this was moved. Have reverted.